src/types/generated/operations.ts

```diff
@@ -1,336 +1,96 @@
-import * as Types from "./types";
+import * as Types from './types';
```

Comment:
did you run fmt or something on this file? not sure why it changed

Reply:
Yep, I generally run `pnpm fmt` on the whole repo.

Reply:
oh I think it ignores `generated` files, not sure why it changed here

Reply:
Hmm maybe `pnpm run indexer-codegen` auto reformats?

Reply:
ohh maybe....

Reply:
Yeah, pnpm run indexer-codegen reformats, maybe we just apply autoformat to even generated files (just ignoring lints of course)